webdriver/tests/bidi/new_session/connect.py

```diff
@@ -1,43 +1,35 @@
 import pytest
 import asyncio
 import websockets
+import webdriver
 
 # classic session to enable bidi capability manually
+# Intended to be the first test in this file
 @pytest.mark.asyncio
 @pytest.mark.capabilities({"webSocketUrl": True})
 async def test_websocket_url_connect(session):
+    assert not isinstance(session, webdriver.BidiSession)
     websocket_url = session.capabilities["webSocketUrl"]
     async with websockets.connect(websocket_url) as websocket:
         await websocket.send("Hello world!")
-        await websocket.close()
 
-# bidi session following classic session to test session
-# recreation with bidi true in session fixture.
-# Close websocket at the end.
+# test bidi_session send
+# using bidi_session is the recommended way to test bidi
 @pytest.mark.asyncio
-@pytest.mark.bidi(True)
-async def test_bidi_session_1(session):
-    await session.websocket_transport.send("test_bidi_session_1")
-    await session.websocket_transport.close()
-
-# bidi session following a bidi session with the same capabilities
-# but closed websocket to test restart of websocket connection.
-@pytest.mark.asyncio
-@pytest.mark.bidi(True)
-async def test_bidi_session_2(session):
-    await session.websocket_transport.send("test_bidi_session_2")
-    await session.websocket_transport.close()
+async def test_bidi_session_send(bidi_session):
+    await bidi_session.websocket_transport.send("test_bidi_session: send")
 
 # bidi session following a bidi session with a different capabilities
 # to test session recreation
 @pytest.mark.asyncio
-@pytest.mark.bidi(True)
 @pytest.mark.capabilities({"acceptInsecureCerts": True})
-async def test_bidi_session_3(session):
-    await session.websocket_transport.send("test_bidi_session_3")
+async def test_bidi_session_with_different_capability(bidi_session):
```

Comment:
As mentioned in my reviews for the initial PR (and what @jgraham also referred to earlier in this PR) lets get an issue filed so we can make sure to have a single test for that. It would most likely need the `new_session` fixture, or maybe a new one for `BiDiSession`.